src/lib/Microsoft.Fx.Portability/Analyzer/PublicKeyToken.cs

```diff
@@ -83,6 +86,9 @@ public override string ToString()
 
         private static ImmutableArray<byte> ParseString(string hex)
         {
+            if (string.Compare(hex, "null", true, CultureInfo.InvariantCulture) ==0)
+                return ImmutableArray<byte>.Empty;
+
             if (hex.Length % 2 != 0)
```

Comment:
Do we also need to check for "actual null" here? Perhaps that should just throw to make diagnostics simpler? Otherwise, we'll nullref here.

Reply:
Looks like the serializer always makes "null" as the string token value when the assembly is not strong named, I added the "actual null" check to return ImmutableArray<byte>.Empty.